src/ads/newsletter-editor/disable-auto-ads.js

```diff
@@ -15,7 +15,7 @@ export default function DisableAutoAds( { saveOnToggle = false } ) {
 		const { getBlocks } = select( 'core/block-editor' );
 		const meta = getEditedPostAttribute( 'meta' );
 		return {
-			disableAutoAds: meta.disable_auto_ads,
+			disableAutoAds: !! meta.disable_auto_ads,
```

Comment:
I snuck this smallfix in as well since I noticed a related error on my local test site when `disable_audo_ads` is an empty string. Forcing this to be a boolean value will ensure this always passes validation.

Let me know if you'd prefer I remove this here and add the fix in another PR.